### PR TITLE
chore: update all docs dependencies (update website banner)

### PR DESCRIPTION
**Pull request by timonrey**
#### Summary
Pull request to update all docs dependencies

#### Description
This pull request updates all docs dependencies. The new `gatsby-theme-docs` update will also update the banner of the website.

### CODE REVIEW

**Review by emmenko — COMMENTED** [review]
Thanks for helping out

File: website/src/content/api-reference/commercetools-frontend-application-components.mdx [hunk 1]
```diff
@@ -132,7 +132,7 @@ const ChannelDeletion = () => {
...
-| `isPrimaryButtonDisabled`       | `boolean`                  |    -     | -                 | false     | Indicates whether the primary button is disabled or not.                                                                                                                                                                                                                                                        |
+| `isPrimaryButtonDisabled`       | `boolean`                  |    -     | -                 | false     | Indicates whether the primary button is deactivated or not.                                                                                                                                                                                                                                                        |
```
[the remark this answers is not in the document]
Comment (nkuehn) [reply website/src/content/api-reference/commercetools-frontend-application-components.mdx:135]: I don't see an issue.  If we want to refer to the specific HTML tag attribute `disabled` it should be formatted as a string literal anyways (like I did here) and is then not part of writing style.  Body text would have to  describe the behavior in other words to explain it and is not good anyways if it just repeats the words in the property name.

File: website/src/content/maintenance-guidelines.mdx [hunk 2]
```diff
@@ -60,7 +60,7 @@ All breaking changes are thoroughly documented in the [release notes](/releases)
 
 # Minimum required maintenance policy
 
-All the native Merchant Center applications are constantly kept up-to-date with the released tools and packages. This ensures that all the latest bug fixes and features are consistently adopted by all applications.
```
[the remark this answers is not in the document]
Comment (nkuehn) [reply website/src/content/maintenance-guidelines.mdx:63]: we can add the specific combination "native Merchant Center application(s)"  to the style rules.  We just did that for "master variant" too, which is OK in that combo but "master" is not in other context.

File: website/src/content/api-reference/commercetools-frontend-application-components.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (emmenko) [reply website/src/content/api-reference/commercetools-frontend-application-components.mdx:135]: Alright, fair enough. Thanks

**Review by emmenko — APPROVED** [review]
Thanks! 🙌

**Pull request merged**